zenoh/src/net/runtime/adminspace.rs

```diff
@@ -307,6 +307,17 @@ impl Primitives for AdminSpace {
             data_info,
         );
 
+        {
+            let conf = self.context.runtime.config.lock();
+            if !*conf.adminspace.changes() {
+                error!(
```

Comment:
prefer `log::error` to `error` to avoid confusion with more aggressive macros and for easier greps